src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -6368,6 +6368,46 @@ IArrayInitializerOperation popAndAssembleArrayInitializerValues(IArrayInitialize
             }
         }
 
+        public override IOperation? VisitCollectionExpression(ICollectionExpressionOperation operation, int? argument)
+        {
+            EvalStackFrame frame = PushStackFrame();
+            var elements = VisitArray(operation.Elements, unwrapElement, wrapElement);
```

Comment:
>wrapElement [](http://example.com/codeflow?start=73&length=11)

nit: it seems `wrapElement` can't be made static (it uses instance method `IsImplicit`). Consider using a lambda around it to cache the allocated delegate.

Reply:
Changed both local functions to lambda expressions.